image_segmentation/pytorch/runtime/training.py

```diff
@@ -63,8 +63,8 @@ def train(flags, model, train_loader, val_loader, loss_fn, score_fn, device, cal
                     metadata={CONSTANTS.FIRST_EPOCH_NUM: epoch, CONSTANTS.EPOCH_COUNT: 1})
         mllog_start(key=CONSTANTS.EPOCH_START, metadata={CONSTANTS.EPOCH_NUM: epoch}, sync=False)
 
-        if is_distributed:
-            train_loader.sampler.set_epoch(epoch)
+        # if is_distributed:
```

Comment:
I do not know what this change is doing, other than that, looks good.

Reply:
Since we don't use distributed samplers this is no longer needed. I commented it out in case someone wanted to revert previous changes and run it as distributed. It doesn't change anything.